src/Cli/func/Helpers/ResolveBuildOptionHelper.cs

```diff
@@ -33,6 +30,17 @@ public static BuildOption ResolveBuildOption(BuildOption currentBuildOption, Wor
                     FileSystemHelpers.FileExists(Constants.RequirementsTxt) &&
                     new FileInfo(Path.Combine(Environment.CurrentDirectory, Constants.RequirementsTxt)).Length > 0)
                 {
+                    // Include default (local) build option for Windows as some customers may be using a local build for testing purposes
+                    // Note that this will be deprecated in the future after 4.1.1 is released
+                    if (includeLocalBuildForWindows && OperatingSystem.IsWindows())
+                    {
+                        ColoredConsole.WriteLine("Python runtime detected on Windows. Using local build option.");
+                        ColoredConsole.WriteLine(OutputTheme.WarningColor($"The default build option for python function apps with a valid requirements.txt " +
+                            $"will be switched over to use remote builds after version 4.1.1.\n" +
+                            "If a local build is still needed, please use the `--build-local` flag when running `func pack`."));
+                        return BuildOption.Default;
```

Comment:
I'm not sure about this messaging and the discoverability of it. @mattchenderson do you have any thoughts or ideas on this?

Reply:
As discussed in our meeting, this is definitely a breaking change, and we should reconsider this. Customers which have an existing CI/CD split between `func pack` and az cli deployment that upgrade their core tools version and leave their build steps as is will see deployment fail, as they will not be passing in the remote build flag to az cli.